File: examples/polyglot-app/ruby/customer_controller.rb
class CustomerController
  attr_accessor :rut, :email, :phone

  def create
    payload = params.require(:customer).permit(:rut, :email, :address, :birthDate)
    Rails.logger.info(payload[:email])
    Faraday.post("https://analytics.example.com/events", payload)
  end
end

File: examples/polyglot-app/php/CustomerController.php
<?php

use Illuminate\Support\Facades\Log;

class CustomerController
{
    protected $fillable = ['rut', 'email', 'phone', 'accountNumber'];

    public function store($request)
    {
        $email = $request->input('email');
        $phone = $_POST['phone'];
        Log::info($email);
        Http::post('https://billing.example.com/customers', ['email' => $email, 'phone' => $phone]);
    }
}
